CRAN-RELEASE

```diff
@@ -0,0 +1,2 @@
+This package was submitted to CRAN on 2019-12-17.
+Once it is accepted, delete this file and tag the release (commit ebf7bcc9db).
```

Comment:
Why do we need this file now?

Reply:
we don't. I added it by accident. I removed it now again